meshroom/core/node.py

```diff
@@ -142,6 +237,7 @@ def __init__(self, node, range, parent=None):
         super(NodeChunk, self).__init__(parent)
         self.node = node
         self.range = range
+        self.log = LogManager(self)
```

Comment:
```
    def __init__(self, node, range, parent=None):
        ...
        self.logManager = LogManager(self)
        ...

    @property
    def logging(self):
        return self.logManager.logger
```

Reply:
I have called the property `logger` instead of `logging` because it makes more sense imo.

Reply:
yes, I agree.